modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -178,4 +193,17 @@ html#{$cadmin-selector} {
 			padding-right: $sidebarWidth;
 		}
 	}
+
+	.categories-popover {
```

Comment:
Maybe we can add this style here to separate the popover from the trigger:
`transform: translateY(-8px);`

Reply:
Also, is it a good idea to talk about this with Clay team? 🤔

Reply:
Not that easy, I see 🤐